packages/mobile/firebase.json

```diff
@@ -0,0 +1,5 @@
+{
+  "react-native": {
+    "messaging_ios_auto_register_for_remote_messages": false
```

Comment:
This is because we don't want the system popup with asking for push permission right when the user runs the app for the first time.
Push permission is asked later on when we create the user account.